update readme to point to pip install instead of source (#139)
SBC-Utrecht/pytom-match-pick: README.md

conda activate pytom_tm
```

Then install the code with `pip` (building cupy can take a while!):

```commandline
python -m pip install pytom-match-pick[plotting]
```

The installation above also adds the optional dependencies `[matplotlib, seaborn]` which are required to run 
`pytom_estimate_roc.py`. They are not essential to the core template matching functionality, so for some systems 
(such as certain cluster environments) it might be desirable to skip them. In that case remove `[plotting]` from the pip install command:

```commandline
python -m pip install pytom-match-pick
```

## Cupy warning

> Trying to import the code with 

Reviewer: ```suggestion
Having issues running the software? If cupy is not correctly installed, 
```

```commandline
python -c "import pytom_tm"
```

> can show a cupy warning. If this is the case, this probably means cupy is not correctly installed.

Reviewer: ```suggestion
will show an import error. If this is the case, this probably means cupy is not correctly installed.

```

Author: Currently the code only does logging.warning, not an actually import error

Reviewer: Fair!

> If your cupy installation is broken, the following should raise some errors:

Reviewer: ```suggestion
Alternatively, cupy can sometimes be installed without issue but not detect CUDA correctly. In that case, the following should raise some errors:
```
